web/settings/smarthomeconfig.php

```diff
@@ -621,6 +621,24 @@
 								</div>
 							</div>
 							<hr class="border-secondary">
+							<div class="form-group">
+							<div class="form-row mb-1">
+							  <label class="col-md-4 col-form-label">Steuerung über Smart Button</label>
+							  <div class="col">
+							    <select class="form-control" name="device_pbtype" id="device_pbtypeDevices<?php echo $devicenum; ?>" data-default="none" data-topicprefix="openWB/config/get/SmartHome/" data-topicsubgroup="Devices/<?php echo $devicenum; ?>/">
+							      <option value="none" data-option="none" selected="selected">Kein Button</option>
+							      <option value="shellypb" data-option="shellypb">Shelly Button 1</option>
+							    </select>
+							  </div>
+							</div>
+							  <div class="form-row mb-1">
```

Comment:
Wie in Deinem Kommentar bereits geschrieben, die Eingabe bitte nur einblenden, wenn der Button im Auswahlfeld auch aktiv ist.

Reply:
gemacht